installation_tips/full_spikeinterface_environement_rolling_updates.yml

```
@@ -0,0 +1,35 @@
+name: si_env
```

```suggestion
name: si_env_rolling
```

Do you want to clobber the name for the installation_tips env? I have a feeling people might try to install both of these separately.... But maybe we force them to figure out how to use conda envs, but I can imagine this appearing on the issue tracker....